pkg/broker/ttl.go

```diff
@@ -26,6 +28,18 @@ const (
 	V02TTLAttribute = "knativebrokerttl"
```

Comment:
Can this be private?

Reply:
can we change that at head and leave the backport as is?

Reply:
:+1: to private and also :+1: to changing it elsewhere.

Reply:
The constant can't be made private as is, it's used in broker ingress currently. (we should change that to use the Function)